src/Day05.kt

```diff
@@ -33,12 +32,11 @@ fun main() {
     fun part2(input: List<String>): Int {
 
         // Criteria for nice list
-        val pattern = Regex("(..).*\\1")
+        val pattern = Regex("(..).*\\1") 
         fun String.equalEnds() = this.windowed(3).any { it.first() == it.last() }
-        fun String.checkForPairs() = this.windowed(2).count() > this.windowed(2).distinct().count()
 
```

Comment:
This is actually quite nice - I like this insight on the comparison between the `count` and `distinct` functions. Good find. 

That said, you don't actually need this method, since the RegEx patterns covers the same thing. You could cut one or the other - doesn't really matter - but you definitely don't need both!